src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -357,6 +357,17 @@ def _apply_type(cls, non_typed_value: str) -> List[BetaBuildInfo]:
 
             return beta_build_infos
 
+    class DeviceUdidsArgument(cli.EnvironmentArgumentValue[List[str]]):
+        environment_variable_key = "DEVICE_UDIDS"
+        example_value = "00000000-000000000000001E"
+
+        @classmethod
+        def _apply_type(cls, non_typed_value: str) -> List[str]:
+            udids = non_typed_value.split()
```

Comment:
`shlex.split` would be safer to use. Better yet, filter out empty strings after stripping them.